ui/src/message_popup/mobile_prototype/message_components.jsx

```diff
@@ -0,0 +1,178 @@
+/* @flow weak */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import _ from 'lodash';
+
+import Paper from 'material-ui/Paper';
+import IconButton from 'material-ui/IconButton';
+
+import FaceIcon from 'material-ui/svg-icons/action/face';
+import InfoOutlineIcon from 'material-ui/svg-icons/action/info-outline';
+
+const Message = React.createClass({
```

Comment:
This is a great use of the `Message` component as a small piece that other components use. 👍 

If you choose to have multiple components in a file like this, please add a quick comment at the top of the file about what this file contains and what's going on.